assets/prototype/infrastructure/services/contextProviders/ConfigProvider.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External imports
+ */
+import { createContext } from '@wordpress/element';
+import { pick, propOr } from 'ramda';
+import { useQuery } from '@apollo/react-hooks';
+
+import { GET_GENERAL_SETTINGS } from '../../../domain/eventEditor/containers/queries/generalSettings';
+import { GET_CURRENT_USER } from '../../../domain/eventEditor/containers/queries/currentUser';
+
+export const ConfigContext = createContext();
+
+const ConfigProvider = ({ children }) => {
+	const { data: generalSettingsData, error: generalSettingsError, loading: generalSettingsLoading } = useQuery(
+		GET_GENERAL_SETTINGS
+	);
+
+	const { data: currentUserData, error: currentUserError, loading: currentUserLoading } = useQuery(GET_CURRENT_USER);
+
+	/**
+	 * To be updated (if we need it) according to this implementation https://github.com/eventespresso/event-espresso-core/pull/1974
+	 */
+	console.log('ConfigProvider', generalSettingsError, generalSettingsLoading, currentUserError, currentUserLoading);
+
+	const generalSettings = propOr({}, 'generalSettings', generalSettingsData);
+	const currentUser = propOr({}, 'viewer', currentUserData);
+	const generalSettingsProps = pick(['dateFormat', 'timezone'], generalSettings);
```

Comment:
The reason I've used `pick` is because of this `_typename`:
![image](https://user-images.githubusercontent.com/1477453/70347426-2afbd000-1869-11ea-8ee1-54bba2128a4f.png)

Reply:
maybe not the best idea, open to suggestions